CONTRIBUTING.md

```diff
@@ -64,6 +65,32 @@ issue yourself. This gives the team visibility of the work you plan to do.
 The other advantage to raising the issue at the outset is that our process requires an
 issue to be associated with every PR (see [patch format](#patch-format)).
 
+### Before submitting a PR
+
+Kata Containers utilizes a number of [CI systems](https://en.wikipedia.org/wiki/Continuous_integration)
+to automatically check every PR. As well as performing execution tests locally on
+your PR before submission, you are also strongly encouraged to run the suite of
+[static checks](https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh)
```

Comment:
I totally agree that we need this sort of helpful doc but I'm wondering how much should be here and how much should be in https://github.com/kata-containers/tests/blob/master/.ci/README.md (with a link to it from here)?

It might be worth mentioning that you can now run `.ci/static-checks.sh -h`as there are a few options (which can help speed up debugging as you can run just a single category of checks if you wish).